android/src/main/java/com/swmansion/gesturehandler/core/StylusData.kt

```diff
@@ -0,0 +1,9 @@
+package com.swmansion.gesturehandler.core
+
+data class StylusData(
+  var tiltX: Double,
+  var tiltY: Double,
+  var altitudeAngle: Double,
+  var azimuthAngle: Double,
+  var pressure: Double
```

Comment:
```suggestion
  val tiltX: Double,
  val tiltY: Double,
  val altitudeAngle: Double,
  val azimuthAngle: Double,
  val pressure: Double
```

Reply:
Done in [b896f2e](https://github.com/software-mansion/react-native-gesture-handler/pull/3111/commits/b896f2edc7161176ecad6b9d7fab165229be139b)